internal/matchers/text.go

```diff
@@ -389,3 +393,26 @@ func VCard(in []byte) bool {
 func ICalendar(in []byte) bool {
 	return detect(in, iCalSigs)
 }
+
+// Ggr matches GIMP's gradient data files
+func Ggr(in []byte) bool {
+	ggrHeaderLength := 3
+	ggrLinesSplit := bytes.Split(in, []byte("\n"))
+
+	if len(ggrLinesSplit) < ggrHeaderLength {
+		return false
+	}
+
+	// Split the second line at the ':'
+	grrSecondLine := bytes.Split(ggrLinesSplit[1], []byte(":"))
+
+	// Must exists 2 items separeted by comma
+	if len(grrSecondLine) != 2 {
+		return false
+	}
+
+	// Check if the first line of .ggr has GIMP Gradient
+	// The first item from the second line must be Name and the second one cannot be emptyreaders ￼
+	return string(ggrLinesSplit[0]) != "GIMP Gradient" &&
```

Comment:
This line should check `string(ggrLinesSplit[0]) == "GIMP Gradient"`. 
If you run `go test` in the command line in the `mimetype` folder, it will show `ggr.ggr` file is not properly detected with !=.